tasks/main.yml

```
@@ -84,11 +50,30 @@
     group: "root"
     state: "link"
 
-- name: "Build containers with docker-compose"
-  community.docker.docker_compose:
-    project_name: "rsd"
-    project_src: "{{ rsd_working_directory }}"
-    state: "present"
-    build: yes
-  tags:
-    - "molecule-idempotence-notest"
+- name: "Start RSD-as-a-service"
+  tags: "molecule-notest"
+  block:
+
+    - name: "Start all containers"
+      community.docker.docker_compose:
+        project_name: "rsd"
+        project_src: "{{ rsd_working_directory }}"
+        state: "present"
+        debug: "yes"
+
+  rescue:
+    - name: "Remove containers and volumes"
+      community.docker.docker_compose:
+        project_name: "rsd"
+        project_src: "{{ rsd_working_directory }}"
+        state: "absent"
+        remove_volumes: "yes"
+      ignore_errors: "yes"
```

ansible-lint found an issue with this task. Unfortunately, the CI job did not fail.

```suggestion
      ignore_errors: "yes"  # noqa ignore-errors
```

is probably required.

---

Thanks for spotting this. It is not necessary to ignore errors here, so I removed this option.